res/controllers/common-hid-packet-parser.js

```diff
@@ -1,1549 +1,1800 @@
 /* global controller */
 
-/** Common HID script debugging function. Just to get logging with 'HID' prefix. */
+/**
+ * Common HID script debugging function. Just to get logging with 'HID' prefix.
+ *
+ * @param {string} message Message to be printed on controller debug console output
+ */
```

Comment:
Should I replace the use of HIDDebug by console.log accross this library? IMHO the prefix HID is not needed, since the name of thecontroller is printed before each message.

Reply:
I replaced all calls of HIDDebug by `console.error`, `console.warn` or `console.log` in the parser itself.